### PR TITLE
4_Shine

### PR DESCRIPTION
**Pull request by shinevue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,5 +87,6 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 ## Contributors
 
 - [John Doe](./john-doe/README.md)
+- [Lucas Marvel](./lucas/README.md)
```
Comment (oddcommits) [inline README.md:90]: you name should be after `Seb` or `White`

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (shinevue) [reply README.md]: I fixed it

**Comment by shinevue** [comment]
I fixed readme

File: john-doe/README.md
Comment (oddcommits) [inline john-doe/README.md:1]: don't remove a sample folder!

**Review by oddcommits — CHANGES_REQUESTED** [review]

File: john-doe/README.md
Comment (shinevue) [reply john-doe/README.md:1]: I come back it.

**Comment by oddcommits** [comment]
resolve conflicts, @shinevue

**Comment by shinevue** [comment]
I fixed comflict. Thanks

**Review by oddcommits — APPROVED** [review]
good job!

**Pull request merged**